Fix command and update text (#4)
supercid/wiki-docs: addons/cmp/installing.md

You might need to change file permissions or ownership of the generated files after the installation.

> After running the commands above you can login to the store admin. You will find settings for Category personalization and merchandizing module under Stores > Configuration > Services > Nosto CMP.

Reviewer: ```suggestion
After running the commands above, you can log-in to the store admin. You will find settings for `Category personalization and merchandizing` module under Stores > Configuration > Services > Nosto CMP.
```
